src/workflow-engine/charms/temporal-worker/oci_factory/activities/activity_consume_events.py

```diff
@@ -4,6 +4,13 @@
 from oci_factory.activities.consumer.config import Config
 from oci_factory.activities.consumer.schema import SchemaClient
 
+import logging
+import os
+import subprocess
+
+# `TWC_LOG_LEVEL` is the mapped value of `log-level` in the charm config
```

Comment:
It is just the `log-level` in the config file. However, since the workflow runs in a container, the `log-level` is mapped to the `TWC_LOG_LEVEL` environmental variable therein.